jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -906,10 +906,12 @@ properties:
           primary_type:
             title: Primary dither pattern type
             type: string
-            enum: [2-POINT, 2-POINT-NOD, 4-POINT, 4-POINT-NOD, 4-POINT-DITHER,
-              ALONG-SLIT-NOD, CYCLING, EXTENDED_TARGET, FULL, FULL-TIGHT, GAUSSIAN,
-              INTRAMODULE, INTRASCA, MAPPING, NONE, POINT_SOURCE, REULEAUX,
-              SPARSE-CYCLING, WFSC]
+            enum: [2-POINT, 2-POINT-NOD, 3-POINT-NOD,
+              4-POINT, 4-POINT-NOD, 4-POINT-DITHER, 4-POINT-SETS,
+              5-POINT-NOD, 5-POINT-SMALL-GRID, 9-POINT-SMALL-GRID,
+              ALONG-SLIT-NOD, CYCLING, EXTENDED-SOURCE, FULL, FULL-TIGHT,
+              GAUSSIAN, INTRAMODULE, INTRASCA, MAPPING, NONE, POINT-SOURCE,
+              REULEAUX, SPARSE-CYCLING, WFSC]
```

Comment:
Should `UNKNOWN` be added to the schema? If this is the source type wouldn't opening an image fail? Is there a test using `UNKNOWN`?